### ISSUE TITLE
Train/eval parameters to be supplied to batch-norm and dropout layers

### ISSUE THREAD

**Issue by 9autilus** [govindnh4cl/squeezenet#35]
Right now, the batch normalization and dropout layer may not be getting the train-or-not parameter that they need for functioning.
